src/ObjectUtils/PdbFile.cpp

```diff
@@ -49,8 +51,11 @@ class SymbolInfoVisitor : public llvm::codeview::SymbolVisitorCallbacks {
     SymbolInfo symbol_info;
     symbol_info.set_name(proc.Name.str());
     symbol_info.set_demangled_name(llvm::demangle(proc.Name.str()));
-    // This address is a relative virtual address (RVA).
-    symbol_info.set_address(proc.CodeOffset);
+    // The address in PDB files is a relative virtual address (RVA), to make the address compatible
+    // with how we do the computation, we need to add both the load bias (ImageBase for COFF) and
+    // the offset of the executable section.
+    symbol_info.set_address(proc.CodeOffset + object_file_info_.load_bias +
+                            object_file_info_.executable_segment_offset);
```

Comment:
so CodeOffset is offset inside .text section?

Reply:
Yes, that's correct.